### DISCUSSION TITLE
tap action

### DISCUSSION THREAD

**Discussion by home409ca** [totaldebug/atomic-calendar-revive#1389] [Feature Requests]
Hi.
Can you add tap action?

**Comment by nehal51086** [comment]
i second this, 
end game: I have a short list view on my main dashboard, would like to tap that card to open the full calander view i have set up on a second page.

**Comment by marksie1988** [comment]
I previously looked at tap action, something I would like to add but my attempts at it failed as I don't know enough about it. 

When I get some time I will look at it again but I'm rather busy at the moment so haven't had the time.

**Reply by boyi01** [reply]
i found a solution, to get the action:

https://community.home-assistant.io/t/gauge-card-tap-action-solution-sharing/713303

`type: custom:button-card
custom_fields:
  gauge:
    card:
      type: custom:atomic-calendar-revive
      enableModeChange: true
      entities:
        - entity: calendar.drescher_becker_gmail_com
          icon: ""
      dimFinishedEvents: true
styles:
  custom_fields:
    gauge:
      - overflow: unset
      - z-index: -1
  card:
    - overflow: unset
    - filter: opacity(100%)
    - padding: 0px
  grid:
    - grid-template-areas: "\"gauge\""
    - grid-template-columns: 100%
    - grid-template-rows: 100%
tap_action:
  action: navigate
  navigation_path: /dashboard-default/energy`

**Comment by marksie1988** [comment]
This has been added in the latest beta
